src/types/nnsDapp.ts

```diff
@@ -0,0 +1,22 @@
+import { Agent } from "@dfinity/agent";
```

Comment:
- do we really want to add and expose nns dapp canister within nns-js? @bitdivine any thought here?

again the nns-js lib is a lib that can by use by the community too.

- goodie: `nnsDapp.ts` -> `nns_dapp.ts` to follow other canister declaration file names

Reply:
I believe that nns-js should ideally be N separate projects, one per back-end canister.  I hope and expect that that will happen eventually.  For now nns-js is a bucket, which I dislike, but I don't have a strong reason for excluding the nns-dapp backend API from that bucket.  I do believe that we should structure the code in the nns-dapp in such a way that we can pull the canisters apart fairly easily.

Reply:
I like the idea of a package per canister. Easier to reason about, to read and maintain? Then all the NNS dapp specific functionality we keep in the nns-dapp repo.

We might want some kind of "ts" module also in our svelte app. So that we can easily reuse it also for the next application.